libraries/src/Mail/MailTemplate.php

```diff
@@ -464,4 +475,28 @@ public static function deleteTemplate($key)
 
 		return $db->execute();
 	}
+
+	/**
+	 * Check and if necessary fix the file name of an attachment
+	 *
+	 * @param   string  $file  Path to the file to be attached
+	 * @param   string  $name  The file name to be used for the attachment
+	 *
+	 * @return  string  The corrected file name for the attachment
+	 *
+	 * @since   4.0.0
+	 */
+	protected function getAttachmentName($file, $name)
```

Comment:
```suggestion
	protected function getAttachmentName(string $file, string $name): string
```

As this is a new method, lets type hint it all the way :)